static/templates/user_profile_modal.hbs

```diff
@@ -67,23 +67,19 @@
                 </div>
             </div>
 
-            <div class="tabcontent" id="groups">
-                <div id="groups-list-section" class="list-section">
-                    <ul>
-                        {{#each groups_of_user}}
-                        <li>{{this.name}}</li>
-                        {{/each}}
-                    </ul>
+            <div class="subscription-group-list" data-simplebar>
+                <div class="tabcontent" id="groups">
+                    <div id="groups-list-section" class="list-section">
+                        <ul>
+                            {{#each groups_of_user}}
+                            <li>{{this.name}}</li>
+                            {{/each}}
+                        </ul>
+                    </div>
                 </div>
-            </div>
 
-            <div class="tabcontent" id="streams">
-                <div id="streams-list-section" class="list-section">
-                    <ul>
-                        {{#each user_streams}}
-                        <li>{{this.name}}</li>
-                        {{/each}}
-                    </ul>
+                <div class="tabcontent" id="streams">
+                    <table class="user-stream-list" data-empty="No stream subscriptions."></table>
```

Comment:
These `data-empty` values should always be tagged for translation.

Reply:
Updated it.